imitation/scripts/config/common.py

```diff
@@ -0,0 +1,7 @@
+DEFAULT_BLANK_POLICY_KWARGS = dict(
+    learning_rate=3e-4,
+    nminibatches=32,
+    noptepochs=10,
+    # WARNING: this is actually 8*256=2048 due to 8 vector environments
+    n_steps=256,
```

Comment:
In `adversarial-policies`, I instead introduced my own `batch_size` variable, and then computed `n_steps = batch_size / num_envs`. It made the config easier, but did have the downside of needing to write an adapter for each Stable Baselines policy. Something worth considering if this becomes painful.